### ISSUE TITLE
[2.4 PTR] Timbal's Focusing Crystal not proccing health or mana through Vampiric Embrace & Touch

### ISSUE THREAD

**Issue by adambcurrier** [Atlantiss/NetherwingBugtracker#9043]
Timbal's Focusing Crystal is not proccing a heal or mana return through Vampiric Embrace or Vampiric Touch.

![image](https://github.com/Atlantiss/NetherwingBugtracker/assets/108972597/6356fca2-91f4-4490-b3ec-50b56be4a4ef)

It should. There are no limited spell lists for those two effects. Here's a TBCC log where the trinket proc is generating health and mana.

https://classic.warcraftlogs.com/reports/vKXFdMZ4HgGrRzCP#fight=21&view=events&source=6

```
00:17.753	Skiq Shadow Bolt Brutallus 739
	
00:17.770	Skiq Vampiric Embrace Skiq Tick +0 (O: 111)
	
00:17.770	Skiq Vampiric Embrace Beepingnoise Tick +0 (O: 110)
	
00:17.770	Skiq Vampiric Embrace Askah Tick +0 (O: 110)
	
00:17.770	Skiq Vampiric Embrace Squiggies Tick +0 (O: 111)
	
00:17.770	Skiq Vampiric Embrace Scx Tick +0 (O: 111)
	
00:17.770	Beepingnoise gains 37 Mana from Skiq's Vampiric Touch
	
00:17.770	Askah gains 37 Mana from Skiq's Vampiric Touch
	
00:17.770	Squiggies gains 37 Mana from Skiq's Vampiric Touch
	
00:17.770	Scx gains 37 Mana from Skiq's Vampiric Touch
	
00:17.770	Skiq gains 37 Mana from Skiq's Vampiric Touch

```
^ With VE at a 15% heal and VT at 5% mana return 739 damage would be a 110.85 heal and 36.95 mana.
	
```
01:06.291	Skiq Shadow Bolt Brutallus 533
	
01:06.308	Skiq Vampiric Embrace Skiq Tick +80
	
01:06.308	Skiq Vampiric Embrace Beepingnoise Tick +80
	
01:06.308	Skiq Vampiric Embrace Askah Tick +80
	
01:06.308	Skiq Vampiric Embrace Squiggies Tick +80
	
01:06.308	Skiq Vampiric Embrace Scx Tick +0 (O: 80)
	
01:06.308	Beepingnoise gains 27 Mana from Skiq's Vampiric Touch
	
01:06.308	Askah gains 27 Mana from Skiq's Vampiric Touch
	
01:06.308	Squiggies gains 26 Mana from Skiq's Vampiric Touch
	
01:06.308	Scx gains 26 Mana from Skiq's Vampiric Touch
	
01:06.308	Skiq gains 27 Mana from Skiq's Vampiric Touch
```

^ 533 damage would be a 79.95 heal and 26.65 mana.

**Comment by buza-me** [comment]
https://www.wowhead.com/tbc/spell=45054/augment-pain
Arcane spell school, shadow priest bonuses should not work

**Comment by adambcurrier** [comment]
> https://www.wowhead.com/tbc/spell=45054/augment-pain Arcane spell school, shadow priest bonuses should not work

"Sends a shadowy bolt at the enemy causing 285 to 475 Shadow damage." https://www.wowhead.com/tbc/spell=45055/shadow-bolt#triggered-by

I dunno how else to reply to being told that the shadow bolts being fired from this trinket are actually arcane. The proc damage already benefits from Shadowform (at least on the PTR, don't have the trinket on live) because it is a shadow spell. There is a well documented comment history (https://www.wowhead.com/item=34470/timbals-focusing-crystal#comments:id=260648 & https://web.archive.org/web/20080406204327/http://www.shadowpriest.com/viewtopic.php?t=11800 & https://web.archive.org/web/20120207232724/http://elitistjerks.com/f77/t16977-tbc_shadow_priest_101_how_melt_faces_effectively/) of this trinket benefiting from most shadow priest talents and debuffs on retail, on top of debuffs like Curse of Elements and the Improved Shadow Bolt debuff (without consuming charges).

As you can see from the logs I linked the trinket proc did interact with VE & VT on TBC Classic.

Yes, these shadow priest bonuses should work.

**Comment by buza-me** [comment]
Yeah my bad I should have clicked deeper. I guess it is the rng generator spell that is arcane, but it triggers shadow bolts with shadow damage.

**Comment by Godmatik** [comment]
Rev 4020 - still an issue
